x-pack/plugins/upgrade_checkup/server/lib/es_migration_apis.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  AssistanceAPIResponse,
+  CallClusterWithRequest,
+  DeprecationAPIResponse,
+  DeprecationInfo,
+  Request,
+} from 'src/core_plugins/elasticsearch';
+
+import fakeAssistance from './fake_assistance.json';
+import fakeDeprecations from './fake_deprecations.json';
+
+export interface EnrichedDeprecationInfo extends DeprecationInfo {
+  index?: string;
+  node?: string;
+  uiButtons: Array<{
+    label: string;
+    url: string;
+  }>;
+}
+
+export interface UpgradeCheckupStatus {
+  cluster: EnrichedDeprecationInfo[];
+  nodes: EnrichedDeprecationInfo[];
+  indices: EnrichedDeprecationInfo[];
+}
+
+export async function getUpgradeCheckupStatus(
+  callWithRequest: CallClusterWithRequest,
+  req: Request,
+  basePath: string
+): Promise<UpgradeCheckupStatus> {
+  // const migrationAssistanceReq = callWithRequest(req, 'transport.request', {
+  //   path: '/_xpack/migration/assistance',
+  //   method: 'GET',
+  // });
+
+  // const deprecationsReq = callWithRequest(req, 'transport.request', {
+  //   path: '/_xpack/migration/deprecations',
+  //   method: 'GET',
+  // });
+
+  // const [migrationAssistance, deprecations] = await Promise.all([
+  //   migrationAssistanceReq,
+  //   deprecationsReq,
+  // ]);
+
+  // Fake data for now. TODO: Uncomment above and remove fake data
+  const migrationAssistance = _.cloneDeep(fakeAssistance) as AssistanceAPIResponse;
```

Comment:
Need to import lodash.

Reply:
Really confused on how this import got removed...apparently tslint sees lodash as a global?